tests/test_gpu_common.py

```diff
@@ -1,3 +1,5 @@
+import cloudpickle as pickle
```

Comment:
We might want to do both `picklle` and `cloudpickle` as parameters for all pickling tests

Reply:
AFAICT, cloudpickle >>> pickle though

Anyway, how would you implement it? Thinking about it, and it doesn't sound obvious... does it?